applications/GeoMechanicsApplication/tests/cpp_tests/test_stress_strain_utitlities.cpp

```diff
@@ -0,0 +1,113 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//
+
+#include "custom_utilities/math_utilities.hpp"
+#include "custom_utilities/stress_strain_utilities.h"
+#include "testing/testing.h"
+#include <boost/numeric/ublas/assignment.hpp>
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(CheckCalculateTrace, KratosGeoMechanicsFastSuite)
+{
+    Vector eye = MathUtils<double>::StressTensorToVector(IdentityMatrix(3));
```

Comment:
This one can be const (same with the eye vector and matrix in the next 2 tests)
```suggestion
    const Vector eye = MathUtils<double>::StressTensorToVector(IdentityMatrix(3));
```

Reply:
Done